src/AdminConsole/Services/PasswordlessManagement/IPasswordlessManagementClient.cs

```diff
@@ -20,4 +21,23 @@ public interface IPasswordlessManagementClient
     Task EnabledManuallyGeneratedTokensAsync(string appId, string performedBy);
     Task DisabledManuallyGeneratedTokensAsync(string appId, string performedBy);
     Task<GetAppIdAvailabilityResponse> IsApplicationIdAvailableAsync(GetAppIdAvailabilityRequest request);
+
+    /// <summary>
+    /// Retrieve a list of all attestation types in the FIDO2 MDS.
+    /// </summary>
+    /// <returns></returns>
+    Task<IEnumerable<string>> GetAttestationTypesAsync();
+
+    /// <summary>
+    /// Retrieve a list of all certification statuses in the FIDO2 MDS.
+    /// </summary>
+    /// <returns></returns>
+    Task<IEnumerable<string>> GetCertificationStatusesAsync();
+
+    /// <summary>
+    /// Get a list of all authenticators in the FIDO2 MDS.
+    /// </summary>
+    /// <param name="request"></param>
+    /// <returns></returns>
+    Task<IEnumerable<EntryResponse>> GetMetaDataStatementEntriesAsync(EntriesRequest request);
```

Comment:
Recommendation: return `Task<IReadOnlyCollection<EntryResponse>>` or `Task<IReadOnlyList<EntryResponse>>` so it's easier to work with the data (no double enumeration warnings, etc).